### PR TITLE
Added Keynote Announcement for Naomi Ceder

### PR DESCRIPTION
**Pull request by Ramya0701**
I added a blog for Announcement of Keynote speaker.

### CODE REVIEW

**Comment by Sc0rpi0n101** [comment]
CC: @vipulgupta2048 @sayanchowdhury @palnabarun

File: content/blog/2020-Announcing-Keynote-Speaker-for-PyconIndia-2020-Naomi.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Title:Announcing Keynote Speaker for PyCon India 2020 - Naomi Ceder
+Date: 2020-09-03 17:00
+Author: Ramya K (@Ramya0701)
+Category: 2020
+Slug: 2020-keynote-announcement-naomi-ceder
+Summary: We are happy to announce that **Naomi Ceder** is a Keynote speaker for Pycon India 2020!
```
Comment (Sc0rpi0n101) [inline content/blog/2020-Announcing-Keynote-Speaker-for-PyconIndia-2020-Naomi.md:6]:
```suggestion
Summary: We are happy to announce that **Naomi Ceder** is a Keynote speaker for PyCon India 2020!
```

File: content/blog/2020-Announcing-Keynote-Speaker-for-PyconIndia-2020-Naomi.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+We are happy to announce that **Naomi Ceder** is a Keynote speaker for Pycon India 2020!
```
Comment (Sc0rpi0n101) [inline content/blog/2020-Announcing-Keynote-Speaker-for-PyconIndia-2020-Naomi.md:8]:
```suggestion
We are happy to announce that **Naomi Ceder** is a Keynote speaker for PyCon India 2020!
```

**Comment by Sc0rpi0n101** [comment]
@Ramya0701 can you fix the two PyCon instances there, please?

File: content/blog/2020-Announcing-Keynote-Speaker-for-PyconIndia-2020-Naomi.md (hunk 1 above)
Comment (palnabarun) [inline content/blog/2020-Announcing-Keynote-Speaker-for-PyconIndia-2020-Naomi.md:6]:
```suggestion
Summary: We are happy to announce **Naomi Ceder** as a Keynote speaker for Pycon India 2020!
```

File: content/blog/2020-Announcing-Keynote-Speaker-for-PyconIndia-2020-Naomi.md (hunk 2 above)
Comment (palnabarun) [inline content/blog/2020-Announcing-Keynote-Speaker-for-PyconIndia-2020-Naomi.md:8]:
```suggestion
We are happy to announce **Naomi Ceder** as a Keynote speaker for Pycon India 2020!
```

File: content/blog/2020-Announcing-Keynote-Speaker-for-PyconIndia-2020-Naomi.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+Naomi Ceder earned a Ph.D in Classics several decades ago, but switched from ancient human languages to computer languages sometime in the last century. Since 2001, she has been learning, teaching, writing about, and using Python.
+
+An elected fellow of the [Python Software Foundation](https://www.python.org/psf), Naomi is the immediate past chair of its board of directors. She also speaks internationally about the Python community, and on inclusion and diversity in technology in general.
+
+By day she leads a team of Python programmers for [Dick Blick Art Materials](https://www.dickblick.com/), and in her spare time she enjoys sketching, knitting, and deep philosophical conversations with her dog.
+
+She can be reached at her twitter handle: [@NaomiCeder](https://twitter.com/naomiceder)
```
Comment (palnabarun) [inline content/blog/2020-Announcing-Keynote-Speaker-for-PyconIndia-2020-Naomi.md:16]:
```suggestion
She can be reached at her Twitter handle: [@NaomiCeder](https://twitter.com/naomiceder)
```

**Review by palnabarun — CHANGES_REQUESTED** [review]

**Review by palnabarun — APPROVED** [review]

**Review by Sc0rpi0n101 — APPROVED** [review]

**Pull request merged**